docs/conf.py

```diff
@@ -117,7 +123,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = []
+# html_static_path = []
```

Comment:
Is there a reason why this is commented out here?
I can see that it's not commented out in the ladybug_tools_template:
https://github.com/ladybug-tools/ladybug-tools-template/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/docs/conf.py#L114

Or wait, maybe it is?
https://github.com/ladybug-tools/ladybug-tools-template/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/docs/conf.py#L127

Why do we have both a commented and un-commented version of this line? Could they just be a single un-commented line? Sorry that I didn't catch this in earlier PRs.

Reply:
Good pont @chriswmackey.  Actually, I did look at this same thing today and I was wondering what was the the logic I sued at the time to leave it like this. My take is that many things were changed and tested out and ended up as a leftover. 
My suggestion is that we should uncomment the second path and delete the top one. I can do this to the ladybug_tools_template. Actually I can go back to all the other repos to be consistent if you think makes sense.